pkg/scheduler/core/generic_scheduler.go

```diff
@@ -413,24 +419,23 @@ func (g *genericScheduler) findNodesThatFit(pod *v1.Pod, nodes []*v1.Node) ([]*v
 			}
 			if fits {
```

Comment:
If we re-write this if statement as below, we can change line 377 to `filtered = make([]*v1.Node, numNodesToFind)`. This saves us some memory.

```go
if fits {
	len := atomic.AddInt32(&filteredLen, 1)
	if len > numNodesToFind {
		cancel()
	} else {
		filtered[len - 1] = g.cachedNodeInfoMap[nodeName].Node()		
	}
}
```

Reply:
DONE.